docs/api/web-view-tag.md

```diff
@@ -634,3 +634,17 @@ Fired when a plugin process is crashed.
 ### Event: 'destroyed'
 
 Fired when the WebContents is destroyed.
+
+### Event: 'media-started-playing'
+
+Emitted when media starts playing.
+
+### Event: 'media-paused'
+
+Emitted when media is paused or done playing.
+
+### Event: 'did-change-theme-color'
+
+Emitted when a page's theme color changes. This is usually due to encountering a meta tag:
+
+    <meta name='theme-color' content='#ff0000'>
```

Comment:
We use following way of pasting code so code highlight can work:

<pre>
```html
your code here
```
</pre>